taxonomy-editor-frontend/src/pages/editentry/ListTranslations.jsx

```diff
@@ -21,6 +21,11 @@ const ListTranslations = ({ nodeObject, setNodeObject }) => {
     const [newLanguageCode, setNewLanguageCode] = useState(''); // Used for storing new LC from Dialog
     const [isValidLanguageCode, setisValidLanguageCode] = useState(false); // Used for validating a new LC
 
+    const languageOptions = []
+    Object.keys(IsoCodes).forEach(function(key){
+        languageOptions.push(key)
+    })
+
```

Comment:
Please remove this and refactor the code as I mentioned above in all three files.